client/accounting.go

```diff
@@ -36,13 +38,20 @@ func (x *PrmBalanceGet) SetAccount(id user.ID) {
 // Return errors:
 //   - [ErrMissingAccount]
 //   - [ErrMissingSigner]
-func (c *Client) BalanceGet(ctx context.Context, prm PrmBalanceGet) (accounting.Decimal, error) {
+func (c *Client) BalanceGet(ctx context.Context, prm PrmBalanceGet) (res accounting.Decimal, err error) {
```

Comment:
Named returns considered harmful and they're not really needed here.

Reply:
It's true